test-suite/shared/src/test/scala/org/scalajs/testsuite/javalib/lang/CharacterTest.scala

```diff
@@ -434,9 +434,136 @@ class CharacterTest {
     assertEquals(0xff3b, Character.toLowerCase(0xff3b)) // ［ => ［
   }
 
-  @Test def toUpperCase_compare_char_and_codepoint(): Unit = {
+  @Test def toTitleCase_compare_char_and_codepoint(): Unit = {
     for (ch <- Character.MIN_VALUE to Character.MAX_VALUE)
-      assertEquals(Character.toUpperCase(ch), Character.toUpperCase(ch.toInt).toChar)
+      assertEquals(Character.toTitleCase(ch), Character.toTitleCase(ch.toInt).toChar)
+  }
```

Comment:
There seems to be a rebasing problem here. The second commit *removes* the test for `toUpperCase_compare_char_and_codepoint()`. It should be kept.